src/shared/getComponentOption.js

```diff
@@ -64,6 +66,10 @@ export default function getComponentOption (opts, result = {}) {
 
       return metaObject
     })
+    result.meta = uniqBy(
+      result.meta.reverse(),
```

Comment:
I had to reverse the list to always consider the last meta object as the "true" one